### PR TITLE
En6 74

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/main/java/org/entando/entando/kubernetes/controller/externaldatabase/ExternalDatabaseController.java [hunk 1]
```diff
@@ -60,7 +55,7 @@
         }
         CustomResourceOperationsImpl<ExternalDatabase, ExternalDatabaseList, DoneableExternalDatabase> result = externalDatabasesInMyNamespace(client, externalDatabaseCrd);
         waitFor(10).seconds().orUntil(() -> result.inNamespace("default").list() != null);
-        ready("ExternalDatabaseController");
+        ready("KeycloakServerController");
```
[the remark this answers is not in the document]
Comment (ampie) [reply src/main/java/org/entando/entando/kubernetes/controller/externaldatabase/ExternalDatabaseController.java]: No that was due to some aggressive refactoring by IntelliJ. Luckily it is not essential - it is just used for the readiness check in the Operators' Docker images. I have changed it in a later branch

**Pull request merged**